dev/packages/example/aws-1.0.0/dataset/cloudwatch/manifest.yml

```diff
@@ -12,16 +12,15 @@ streams:
         description: Cloudwatch metrics.
         type: period
         default: 5m
+      - name: metrics
+        # Means a yaml text block
+        type: yaml
+        default: >
+          - namespace: AWS/EC2
```

Comment:
@jen-huang Kind of curious if I got the indentation right for the UI. I assume it should have no space in front of the `-`

Reply:
let's merge and deploy this to see what the json response is like (I don't have this repo built locally yet 🙂)